bigquery/snippets/job/bigquery_copy_table_cmek.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package job
+
+// [START bigquery_copy_table_cmek]
+
+import (
+	"cloud.google.com/go/bigquery"
+	"context"
+	"fmt"
+)
+
+func copyTableWithCMEK(projectID, datasetID, tableID string) error {
+	// projectID := "my-project-id"
+	// datasetID := "mydataset"
+	// tableID := "mytable"
+	ctx := context.Background()
+	client, err := bigquery.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("bigquery.NewClient: %v", err)
+	}
+
+	srcTable := client.DatasetInProject("bigquery-public-data", "samples").Table("shakespeare")
+	copier := client.Dataset(datasetID).Table(tableID).CopierFrom(srcTable)
+	copier.DestinationEncryptionConfig = &bigquery.EncryptionConfig{
+		// TODO: Replace this key with a key you have created in Cloud KMS.
```

Comment:
Perhaps make this an argument instead of a TODO? Everyone will have to replace it.

Reply:
In this case, we've done this explicitly because we've gotten feedback previously that the format of the CMEK reference string wasn't clear.